src/widget/wcuemenupopup.h

```diff
@@ -30,10 +31,9 @@ class WCueMenuPopup : public QWidget {
         }
     }
 
-    void popup(const QPoint& p, QAction* atAction = nullptr) {
-        Q_UNUSED(atAction);
-        qDebug() << "Showing menu at" << p;
-        move(p);
+    void popup(const QPoint& p) {
+        auto parentWidget = dynamic_cast<QWidget*>(parent());
+        move(mixxx::widgethelper::mapPopupToScreen(*parentWidget, p, size()));
```

Comment:
A the result of mapPopupToScreen() to a local variable `popupPosition` might improve readability.